src/Integration.UnitTests/Suppression/SonarQubeIssuesProviderTests.cs

```diff
@@ -92,8 +97,12 @@ public void Constructor_TimerIsInitialized()
             mockTimer.VerifySet(t => t.AutoReset = true, Times.Once);
             VerifyTimerStart(Times.Once());
             timerRunning.Should().Be(true);
-            VerifyServiceIsConnected(Times.Once()); // bound -> check connection status...
```

Comment:
Test was failing intermittently because the background thread hadn't always started